libs/src/api/user.js

```diff
@@ -712,6 +712,11 @@ class Users extends Base {
     }
     let primaryEndpoint = CreatorNode.getPrimary(creatorNodeEndpoint)
     let secondaries = CreatorNode.getSecondaries(creatorNodeEndpoint)
+
+    if (secondaries.length <= 1) {
```

Comment:
I think this should be its own PR

Reply:
good catch, nit - i think it would be more readable / accurate if the check was `< 2`